src/stream/src/executor/sink.rs

```diff
@@ -185,6 +185,9 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
             && !self.sink_param.downstream_pk.is_empty();
         // Don't compact chunk for blackhole sink for better benchmark performance.
         let compact_chunk = !self.sink.is_blackhole();
+        tracing::info!("Sink info: sink_id: {} actor_id: {}, need_advance_delete: {}, re_construct_with_sink_pk: {}",
```

Comment:
Can be `tracing::info!(sink_id, actor_id, need_advanc_delete, re_construct_with_sink_pk, "sink info")` to avoid manually generating the format string.